### PR TITLE
docs: Create architecture diagram

### PR DESCRIPTION
**Pull request by ZJvandeWeg**

### CODE REVIEW

File: docs/contribute/architecture.md [hunk 1]
```diff
@@ -99,3 +99,48 @@ This plugin uses the Node-RED [Authentication API](https://nodered.org/docs/user
...
+        Protocol tcp-tls
+        Port default-5432
+    }
+    MOSQUITTO {
+        Protocol HTTP-TLS-WSS-MQTT
+        Port default-1883
+    }
+    FLOWFORGE-FILE-SERVER {
+        Protocol HTTP-TLS
+        Port default-QUESTION
```
Comment (ZJvandeWeg) [inline docs/contribute/architecture.md:126]:
```suggestion
        Port default-3001
```

File: docs/contribute/architecture.md [hunk 2]
```diff
@@ -99,3 +99,48 @@ This plugin uses the Node-RED [Authentication API](https://nodered.org/docs/user
...
+        Protocol HTTP-TLS
+        Port default-3000
+    }
+    POSTGRESQL {
+        Protocol tcp-tls
+        Port default-5432
+    }
+    MOSQUITTO {
+        Protocol HTTP-TLS-WSS-MQTT
+        Port default-1883
```
Comment (hardillb) [inline docs/contribute/architecture.md:122]:
```suggestion
        Port default-1883
        Port default-1884
```
The broker runs 2 ports:
- 1883 for native MQTT (used by projects inside the cluster)
- 1884 for MQTT over WS which is proxied by nginx/ingress to port 80/443 outside the cluster

File: docs/contribute/architecture.md [hunk 3]
```diff
@@ -99,3 +99,48 @@ This plugin uses the Node-RED [Authentication API](https://nodered.org/docs/user
...
+    }
+    FLOWFORGE-FILE-SERVER {
+        Protocol HTTP-TLS
+        Port default-3001
```
Comment (hardillb) [inline docs/contribute/architecture.md:126]: Only used inside the cluster

**Review by hardillb — COMMENTED** [review]
I'll render this to see how it looks and make some more comments later

**Pull request merged**